opsdroid/connector/matrix/connector.py

```diff
@@ -210,12 +210,10 @@ def filter_json(self):
             except Exception:  # pylint: disable=W0703
                 _LOGGER.exception(_("Matrix sync error."))
 
-    def lookup_target(self, alias):
-        """Convert a room alias to the corresponding room ID."""
-        try:
-            return self.room_ids[alias]
-        except KeyError:
-            return alias
+    def lookup_target(self, room):
+        """Convert name or alias of a room to the corresponding room ID."""
```

Comment:
Let's check that I understand it correctly:
- Room name is a string like `main` which comes from the configuration file.
- Room alias is a Matrix specific thing, like: `#main:matrix.org`
I edited docstring in scope of these.

>Can this also lookup the config names for these rooms?

Yes it can. But, I added a call of `get_roomname`, because when a skill developer constrains a method with `#public:matrix.org` instead of just `main`, it should work too, imo. What do you think? Should I revoke this?

Reply:
Yeah I agree, having both is the way forward.

For clarity, I think the input to `lookup_target` should be able to be any of:

* "name" like `'main'`
* matrix room alias like `#matrix:matrix.org`
* a room id (`lookup_target` is a no-op)